databases/week2/part4.sql

```diff
@@ -0,0 +1,45 @@
+--Part 4: Creating a database
+
+CREATE database students;
+
+use students;
+--student table
+CREATE Table `student` (
+    `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `first_name` VARCHAR (32),
+    `last_name` VARCHAR (64)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+--course table
+CREATE Table `course` (
+    `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR (32),
+    `teacher_id` INT (10)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+--teacher table
+CREATE Table `teacher` (
+    `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR (64),
+    `teacher_id` VARCHAR (64)
```

Comment:
you might want to remove the column 'teacher_id' in teacher table, unless if it is intentional